src/storage/oos_file.cpp

```diff
@@ -0,0 +1,570 @@
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include <cassert>
+#include "error_code.h"
+#include "file_manager.h"
+#include "memory_alloc.h"
+#include "page_buffer.h"
+#include "porting_inline.hpp"
+#include "scope_exit.hpp"
+#include "slotted_page.h"
+#include "storage_common.h"
+#include "page_buffer_support.hpp"
+
+#include "oos_file.hpp"
+#include "oos_log.hpp"
+// XXX: SHOULD BE THE LAST INCLUDE HEADER
+#include "memory_wrapper.hpp"
+
+// ****************************************************************************
+// static functions
+// ****************************************************************************
+
+static const auto_unfix_page_ptr
+oos_find_best_page (THREAD_ENTRY *thread_p, const VFID &oos_vfid, const int rec_length, VPID &vpid);
+
+static int
+oos_vpid_init_new (THREAD_ENTRY *thread_p, PAGE_PTR page, void *args);
+
+static int oos_insert_within_page (THREAD_ENTRY *thread_p, const VFID &oos_vfid, RECDES &recdes,
+				   const OOS_RECORD_HEADER &header, OID &oid);
+static int oos_insert_across_pages (THREAD_ENTRY *thread_p, const VFID &oos_vfid, RECDES &recdes,
+				    OID &oid);
+static int
+oos_read_within_page (THREAD_ENTRY *thread_p, const OID &oid, RECDES &recdes,
+		      OOS_RECORD_HEADER &out_header);
+static int
+oos_read_across_pages (THREAD_ENTRY *thread_p, const OID &oid, RECDES &recdes,
+		       OOS_RECORD_HEADER &out_header);
+
+STATIC_INLINE __attribute__ ((ALWAYS_INLINE)) int oos_get_max_chunk_size_within_page ();
+
+static int get_recently_inserted_oos_vpid (const VFID &oos_vfid, VPID &vpid);
+
+// ****************************************************************************
+// memory map used to store recently inserted OOS VPID for each OOS VFID
+//
+// this is used in oos_find_best_page to quickly find the last inserted page
+// ****************************************************************************
+
+struct VFIDHash
+{
+  std::size_t operator() (const VFID &v) const noexcept
+  {
+    return std::hash<int32_t> {} (v.fileid) ^ (std::hash<short> {} (v.volid) << 1);
+  }
+};
+struct VFIDEq
+{
+  bool operator() (const VFID &a, const VFID &b) const noexcept
+  {
+    return a.fileid == b.fileid && a.volid == b.volid;
+  }
+};
+static std::unordered_map<VFID, VPID, VFIDHash, VFIDEq> oos_recently_inserted_oos_vpid_map;
+
+// ****************************************************************************
+
+using namespace oos_log;
+
+// review point: should it be 8?
+static constexpr int OOS_ALIGNMENT = 8;
+
+
+int
+oos_create (THREAD_ENTRY *thread_p, VFID &oos_vfid)
+{
+  FILE_DESCRIPTORS des; // unused
+  int err = NO_ERROR;
+  err = file_create_with_npages (thread_p, FILE_OOS, 1, &des, &oos_vfid);
+  if (err != NO_ERROR)
+    {
+      oos_error ("file_create_with_npages failed");
+      return err;
+    }
+  return NO_ERROR;
```

Comment:
가독성을 위해 각 주요 블럭 사이에 공백 라인을 추가해 주시는게 좋을 것 같습니다. 구현하신 코드 전반에 대해 반영 검토 요청드립니다.
```suggestion
  FILE_DESCRIPTORS des; // unused
  int err = NO_ERROR;
  
  err = file_create_with_npages (thread_p, FILE_OOS, 1, &des, &oos_vfid);
  if (err != NO_ERROR)
    {
      oos_error ("file_create_with_npages failed");
      return err;
    }
    
  return NO_ERROR;
```

Reply:
<img width="1670" height="1283" alt="image" src="https://github.com/user-attachments/assets/1f6bfdfb-a529-4238-bfcd-9c3b91585aa4" />

처리하였습니다!

주요 함수의 윗 공간을 확보하고, 에러 처리 로직 다음에도 새 줄을 처리하여 각 주요 단계 별로 공백을 확보했습니다.

> 위 사진의 2배 정도 공백 줄 추가